pom.xml

```diff
@@ -2075,7 +2075,7 @@
     <properties>
 
         <!--Carbon Identity Framework Version-->
-        <carbon.identity.framework.version>5.20.342</carbon.identity.framework.version>
+        <carbon.identity.framework.version>5.20.345-SNAPSHOT</carbon.identity.framework.version>
```

Comment:
Version number to be confirmed after merging wso2/carbon-identity-framework#3959